src/app/containers/NewStoryPromo/Promo/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useContext } from 'react';
+import { node, string } from 'prop-types';
+import { ServiceContext } from '#contexts/ServiceContext';
+
+import Timestamp from '#components/Promo/timestamp';
+import LiveLabel from '#legacy/psammead-live-label/src';
+import useViewTracker from '#hooks/useViewTracker';
+import ImageWithPlaceholder from '#app/containers/ImageWithPlaceholder';
+
+import Heading from '../PromoAtoms/Heading';
+import Link from '../PromoAtoms/Link';
+import MediaIndicator from '../PromoAtoms/MediaIndicator';
+import Content from '../PromoAtoms/Content';
+import { PromoWrapper, ContentWrapper } from '../PromoAtoms/Wrappers.styles';
+
+const PromoContext = React.createContext({});
+
+const eventTrackingData = {
+  block: {
+    componentName: 'top-stories',
+  },
+};
+
+const Promo = ({ children, to, id }) => {
+  const { script, service, dir } = useContext(ServiceContext);
```

Comment:
I see the wrapping div for the promo text has a dir attribute - not sure it should if we are appling the new coding standards? If so, is this coming from here and / or line 31?

Reply:
I have not thought about the implementation of RTL and LTR yet. I am not sure how I will approach that, and if I will follow the new standards as may be impossible for me to do right now without the theorised infrastructure.

Reply:
Cool, was just pointing it out

Reply:
I did not need dir prop in the end, therefore removed